javaharness/java/arcs/android/AndroidArcsEnvironment.java

```diff
@@ -41,7 +38,9 @@
   private static final Logger logger =
     Logger.getLogger(AndroidArcsEnvironment.class.getName());
 
-  private static final String ASSETS_PREFIX = "https://$assets/";
+  private static final String ASSETS_MACRO = "https://$assets/";
```

Comment:
Macro? That name is a bit confusing to me. tbh I preferred ASSETS_PREFIX, or maybe ASSETS_PATH.

Reply:
Done.